md/variables.md

```diff
@@ -12,13 +12,29 @@ There are several types of variables:
   * JavaScript Expression
   * External API
   * URL parameter
+* Data Management
+  * Business data
 
 ## Evaluation cycle
 
 String and JSON variables are initialized at page data model instantiation and their values do not change subsequently. 
 
 Expression, external API and URL parameter variables are evaluated every time something happens on the page, such as when receiving data or on a user interaction. The call of an External API variable is triggered every time the URL of the resource it points at changes.
 
+#### Business Data
+
+Business Data variable are used to fetch Business Data Model more easier than with external API. With this type. Just follow the wizard, select the query to retrieve Business Object, use dynamic variable (use `{{ myVariable }}`) or static value for filter parameter and set pagination data. Then you can bind your variable into widget or use it in Javascript Expression. 
```

Comment:
```suggestion
Business Data variables are used to fetch Bonita Business Data easier than with Business Data API.

A Business Data variable can be created in two ways:
- From Data Model section in the left panel, drag and droping a Business Object to the whiteboard area
- From the Variables bottom panel, creating a new variable with the type Business Data

A configuration wizard appears: create a Business Data variable by providing information on how to retrieve elements of this Business Object.

==> Copy <Define how to retrieve Business Object and Use your Business data> from embedded doc here.

```

Reply:
"Business Data variables are used to fetch Bonita Business Data easier than with Business Data API."
...business business business

Reply:
Business Data variables are meant to retrieve BDM objects defined in Studio. These objects are accesible through REST API calls, so they were retrieved using External API variables defining the URL. 
For example, using this URL: 
```
../API/bdm/businessData/{{businessDataType}}?q={{queryName}}&p=0&c=10&f={{filter}}
```
It implied to know by heart:
 ``businessDataType`` As example *com.company.model.Invoice*
 ``queryName`` One of object available queries, as example *findByNumInvoice*
 ``filter`` All filters required by the query, as example *f=numInvoice=XXXX*

With Business Data variables, you are able to select one object (from left panel or select box), see its available queries and select one, and configure its filters which are already proposed.
This allows boost and simplify configuration on these data

Reply:
Interesting details for a Fred that were using previous UID versions.
But for a Cindy that just start using UID, it can be confusing...
I suggest to put this in a dedicated section, such as "Details for users that used previous UI Designer versions"

Reply:
> Business Data variables are meant to retrieve BDM objects defined in Studio. These objects are accesible through REST API calls, so they were retrieved using External API variables defining the URL.

Business Data variables are meant to retrieve BDM objects defined in Studio. These objects are accessible through REST API calls, so they can be retrieved using External API variables defining the URL.

> With Business Data variables, you are able to select one object (from left panel or select box), see its available queries and select one, and configure its filters which are already proposed.
This allows boost and simplify configuration on these data

I would just say that it is not needed anymore with Business Data variables.